Privacy Statement (#106)
fairpm/fair-plugin: README.md

* Media features provided by OpenVerse are disabled, pending discussion and work by the FAIR working group
* Ping services are configured to use IndexNow in place of Pingomatic

### Data Privacy

FAIR is built to reduce external dependencies and keep your site as self-contained as possible. However, some essential features require connecting to remote services in order to function correctly. This section details which features involve remote requests, what data may be transmitted, and the specific third-party providers involved. Review the list below to understand exactly where and why your site may communicate with external endpoints.

* Ping Services when a post is published is handled by [IndexNow](https://www.indexnow.org).

> * The list of WordPress Events shown on your WP Admin Dashboard is powered by [The WP World](https://thewp.world)

Reviewer: “is powered by” insinuates active support of FAIR by them; I’d replace it with “is retrieved from”

```suggestion
* The list of WordPress Events shown on your WP Admin Dashboard is retrieved from [The WP World](https://thewp.world)
```
```suggestion
* The list of WordPress Events shown on your WP Admin Dashboard is powered by [The WP World](https://thewp.world)
```

* Media features provided by OpenVerse are disabled, pending discussion and work by the FAIR working group
* Ping services are configured to use IndexNow in place of Pingomatic

### Data Privacy

* See Also: [Linux Foundation Projects Privacy Policy](https://lfprojects.org/policies/privacy-policy/)

FAIR is built to reduce external dependencies and keep your site as self-contained as possible. However, some essential features require connecting to remote services in order to function correctly. This section details which features involve remote requests, what data may be transmitted, and the specific third-party providers involved. Review the list below to understand exactly where and why your site may communicate with external endpoints.

> * Alerting Ping services when a post is published is handled by [IndexNow](https://www.indexnow.org).

Reviewer 2: Grammar/consistency:

```suggestion
* Search engine pings services when a post is published are handled by [IndexNow](https://www.indexnow.org).
```

> * Installation and updates of all WordPress Packages (core, plugins, themes) are via [AspireCloud from AspirePress](https://aspirepress.org/)

Reviewer 2: ```suggestion
* Installation and updates of all WordPress Packages (core, plugins, themes) are via [AspireCloud from AspirePress](https://aspirepress.org/) (or other mirror as configured)
```

* PHP versions are provided by [php.net](https://php.net)
* Twemoji (emoji assets) are retrieved by [jsDeliver](https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/)

> In addition we self-host certain features that could not be properly protected on our API servers as an intermediary. All data collected from FAIR servers fall under the [Linux Foundation Policies and Terms of Use for Hosted Projects](https://lfprojects.org/policies/hosted-project-tools-terms-of-use/). These services include:

Reviewer 2: ```suggestion
In addition we self-host certain features that could not be properly protected on our API servers as an intermediary. All data collected from FAIR servers fall under the [Linux Foundation Policies and Terms of Use for Hosted Projects](https://lfprojects.org/policies/hosted-project-tools-terms-of-use/); additionally, [the server itself is open source](https://github.com/fairpm/server). These services include:
```
